cfme/configure/access_control.py

```diff
@@ -571,105 +912,120 @@ def tree_path(self):
     def parent_path(self):
         return self.tree_path[:-1]
 
-    def create(self, cancel=False):
```

Comment:
cancel was not used anywhere in the method, removed as unused
Should I restore it?